EIPS/eip-3300.md

```diff
@@ -0,0 +1,66 @@
+
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+On the block this EIP activates, and again every 100 blocks, the gas refunds for `SELFDESTRUCT` and `SSTORE` would diminish by 1, until reaching their refund activation cost.
+When reaching their refund activation cost, refunds will be removed.
+The refund activation cost of `SSTORE` is defined to be the lowest possible opcode cost of setting a nonzero value to zero.
+The refund activation cost of `SELFDETRUCT` is defined to be the lowest possible opcode cost of `SELFDESTRUCT`.
+
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+The refunds would become worthless before they fall below their theoretical activation cost, so they can be safely removed at that point.
+Activation costs are defined this way to be compatible with other EIPs that may change gas costs in the future.
+
+The rate of diminishing specified would currently require (24000-5000) * 100 = 1,900,000 blocks for `SELFDESTRUCT` and (15000-800) * 100 = 1,420,000 blocks for `SSTORE`.
+This timeframe is currently about a year, which should be more than enough flexibility for the remaining refunds to be consumed.
+
+## Backwards Compatibility
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+This proposal breaks gas refunds, which contribute to block elasticity.
+The effect of this will be increased gas price volatility: higher highs and lower lows.
+
+## Test Cases
+<!--Test cases for an implementation are mandatory for EIPs that are affecting consensus changes. Other EIPs can choose to include links to test cases if applicable.-->
+TODO
+
+## Implementation
+<!--The implementations must be completed before any EIP is given status "Final", but it need not be completed before the EIP is accepted. While there is merit to the approach of reaching consensus on the specification and rationale before writing code, the principle of "rough consensus and running code" is still useful when it comes to resolving many discussions of API details.-->
+TODO
+
```

Comment:
```suggestion
```
These are not required sections, recommend leaving them out until such time as you have stuff to put in them.